prep-javascript-functions/main.js

```diff
@@ -0,0 +1,78 @@
+function addTwoNumbers_(x, y) {
+  var result = x + y;
+  return result;
+}
+
+var sum_ = addTwoNumbers_(4, 4);
+console.log('sum_', sum_);
+
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var sum = addTwoNumbers(2, 2);
+console.log('sum', sum);
+
+function convertHoursToMinutes(hours) {
+  var result = hours;
+  return result;
+}
+
+var Minutes = convertHoursToMinutes(2 * 60);
+console.log(Minutes);
+
+function getGreeting(name) {
+  var result = name;
+  return result;
+}
+
+var Hello = getGreeting('"Hello World!"');
+console.log(Hello);
+
+function addAndMultiplyBy5(num1, num2) {
+  var result = (num1 + num2) * 5;
+  return result;
+}
+
+var mutiply = addAndMultiplyBy5(10, 5);
+console.log(mutiply);
+
+function multiplyAndDivideBy5(num1, num2) {
+  var result = (num1 * num2) / 5;
+  return result;
+}
+
+var divide = multiplyAndDivideBy5(35, 10);
+console.log(divide);
+
+function subtractTwoNumbers(num1, num2) {
+  var result = (num1 - num2);
+  return result;
+}
+
+var subtract = subtractTwoNumbers(22, 7);
+console.log(subtract);
+
+function getCircleCircumference(radius) {
+  var result = radius;
+  return result;
+}
+
+var circumfrerence = getCircleCircumference(5 * 2 * Math.PI);
+console.log(circumfrerence);
```

Comment:
For this one we want to move the multiplication into the function's code block so we can pass any radius to get the circumference.